alis/alfunc_gaussian.py

```diff
@@ -94,12 +94,16 @@ def set_vars(self, p, level, mp, ival, wvrng=[0.0,0.0], spid='None', levid=None,
                         if mp['mlnk'][j][0] == mp['mtie'][ival][i]:
                             cmd = 'lnkprm = ' + mp['mlnk'][j][1]
                             exec(cmd)
+                getid = mp['tpar'][mp['mtie'][ival][i]][1]  # THIS IS A JXP KLUDGE.  WHAT IS CORRECT??
```

Comment:
I don't think this line is needed. Going through the various cases, `lnkprm=None` before the `if` loop (on line 85).

(1) The `if` case (line 87): `getid` is set and `lnkprm` is unchanged from None.

(2) The `elif` case (line 89): Further split into an `if/else` case; in both of these, lnkprm is set (on line 91, and then the combination of line 95+96). The only possibility where this might break is if the `if` statement on line 94 is never satisfied in the for loop, which would point to a bug somewhere else in the code.

(3) The `else` case (line 99): `getid` is set, and `lnkprm` is unchanged

Then, on line 102, we only need one of `lnkprm` and `getid` to be set. And this is (should!) be satisfied above.

I think the most likely problem is that the links are not defined correctly in the input model settings file, or the link being attempted is not permitted in the current version of ALIS. If you have an example model and data file, I can probably diagnose the problem a little better (or maybe `exec` is not allowed under python3?)

The bottom line is that this addition to the code shouldn't be here (the bug/problem is due to something else), but thanks for raising the issue!